templates/page_object.rb

```diff
@@ -0,0 +1,7 @@
+class PageObject
```

Comment:
I like Martin Fowler's take on [Page Objects]](http://martinfowler.com/bliki/PageObject.html)).

I've followed his advice:
> Despite the term "page" object, these objects shouldn't usually be built for each page, but rather for the significant elements on a page [1].
and it has allowed me to re-use page objects across feature specs.